Nashua.txt

```diff
@@ -0,0 +1,8 @@
+## Nashua, NH
+
```

Comment:
Oh, I wish I didn't mind  snow. I much prefer the sun, which is kind of strange since I live in Seattle.